src/lib/components/Modal/README.mdx

```diff
@@ -1017,6 +1017,17 @@ opened.
 
 <!-- markdownlint-disable MD024 -->
 
+## Forwarding HTML Attributes
+
+In addition to the options below in the [component's API](#api) section, you
+can specify [React synthetic events] or **any HTML attribute you like.** All
+attributes that don't interfere with the API are forwarded either to the native
+HTML `<div>` or to the `<button>` in case of `ModalCloseButton` component. This
```

Comment:
This would require some modification. For. example HTML tag of  `ModalTitle` can be changed, but it is heading by default. What do you think @adamkudrna ?

Reply:
What more, we need to inform end-user that `Modal` will transfer props to different element than root element

Reply:
Please take a look at changes made.